### ISSUE TITLE
Maintain screen brightness between shutdown-reboot-suspend-resume-hibernate

### ISSUE THREAD

**Issue by leledumbo** [stockmind/gpd-pocket-ubuntu-respin#69]
I have set screen brightness to lowest level in KDE system settings. This is maintained if I reboot or shutdown then resume or power on. However, if I just suspend (e.g. close the lid) then upon resume the brightness level is set to max. I'm thinking of putting a script in /etc/pm/sleep.d executing xbacklight, but if there's anything better (as I have to manually change the script if someday I want different default value) I want to know.

**Comment by stockmind** [comment]
Hi! this issue still persist with latest kernel?

**Comment by leledumbo** [comment]
Yes, in your 4.15.0-rc7 kernel. Petr's kernel 4.15.0 fails to properly suspend (and shutdown) so the issue cannot be reproduced there.
